### ISSUE TITLE
Some String's will not be transformed in Class Files

### ISSUE THREAD

**Issue by c-koell** [eclipse/transformer#286]
Hi!

We have some Code that looks like following
```
public static String extractEventFromRequest(FacesContext facesContext)
  {
    Map params = facesContext.getExternalContext().getRequestParameterMap();
    String behaviorEvent = (String)params.get("javax.faces.behavior.event");
    return behaviorEvent;
  }
```
or 

```
import org.primefaces.PrimeFaces;
import org.primefaces.util.Constants;

public void beforePhase(PhaseEvent event) {
	if (event.getFacesContext().getPartialViewContext().isAjaxRequest()) {
		String src = event.getFacesContext().getExternalContext().getRequestParameterMap().get(Constants.RequestParams.PARTIAL_SOURCE_PARAM);
			...
		}
	}
```
This example will result to following class file
```
public void beforePhase(PhaseEvent event) {
	if (event.getFacesContext().getPartialViewContext().isAjaxRequest()) {
		String src = event.getFacesContext().getExternalContext().getRequestParameterMap().get("javax.faces.source");
			...
		}
	}
```

With 0.4.0 the string is still javax...

**Comment by c-koell** [comment]
A other string that will also not be transfomred is
```
private static String[] wellKnownResourceFolders = { "javax.faces.resource" };
```

**Comment by bjhargrave** [comment]
This is fixed now by #309 which added many new Faces strings.
